example/app.config.js

```diff
@@ -39,4 +39,25 @@ export const app_config = {
    * If enabled, the user will be able to save sessions on their device for later use.
    */
   enableConnectionSessions: true,
+
+  /*
+   * This configuration allows you to activate or not the subtitles under app icons.
+   * If enabled, subtitles will be displayed under app icons to provide additional
+   * context to the user.
+   */
+  showModulesSubtitle: false,
+
+  /*
+   * This configuration allows you to customize the colors used in the application.
+   * You can use the `ThemeColors` object to specify primary, secondary, error, and
+   * other color values.
+   */
+  themeColorsConfig: {},
```

Comment:
Please add here the value of primaryColor, secondaryColor and secondaryColor_dark as example and to make it easier to modify please